src/RefractAPI.cc

```diff
@@ -41,15 +41,18 @@ namespace drafter {
 
     refract::IElement* DataStructureToRefract(const snowcrash::DataStructure& dataStructure)
     {
-        refract::IElement* msonElement = MSONToRefract(dataStructure);
+        refract::IElement *msonElement = NULL, *msonExpanded = NULL;
```

Comment:
This line is needed. Furthermore please do declare variables on two lines without using the comma operator.